api-docs/cloud/swaggerV1Compat.yml

```diff
@@ -97,35 +77,24 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
         '413':
-          description: >-
-            Write has been rejected because the payload is too large. Error
-            message returns max size supported. All data in body was rejected
-            and not written.
+          description: Write has been rejected because the payload is too large. Error message returns max size supported. All data in body was rejected and not written.
```

Comment:
```suggestion
          description: Write has been rejected because the payload is too large. Error message returns maximum size supported. All data in body was rejected and not written.
```